apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/PartitionedReadTests.cs

```diff
@@ -29,42 +31,72 @@ public class PartitionedReadTests
         public PartitionedReadTests(PartitionedReadTableFixture fixture) =>
             _fixture = fixture;
 
-        [CombinatorialData]
-        [Theory]
-        public async Task DistributedReadAsync(bool query)
+        [Theory, CombinatorialData]
+        [Trait(Constants.SupportedOnEmulator, Constants.No)]
+        public async Task DistributedReadAsync(bool query, bool dataBoostEnabled)
```

Comment:
I think this test should be split in two, DistributedReadAsync_Select, DistributedReadAsync_Read.

I understand it was like that before, but test in general shouldn't have conditional setup and/or asserts.

(It's fine if the obsolete one remains as is, as we will remove it)